### PR TITLE
feat(remix-netlify): add support for `@netlify/functions` v0.11

### PR DESCRIPTION
**Pull request by donavon**
The package was recently updated to version 1.0. No changes were made, just the version number bump.

### CODE REVIEW

**Review by MichaelDeBoey — APPROVED** [review]

File: packages/remix-netlify/package.json [hunk 1]
```diff
@@ -15,9 +15,9 @@
     "@remix-run/node": "1.3.2"
   },
   "peerDependencies": {
-    "@netlify/functions": "^0.10.0"
+    "@netlify/functions": "^0.10.0 || ^0.11.0 || ^1.0.0"
   },
   "devDependencies": {
-    "@netlify/functions": "^0.10.0"
+    "@netlify/functions": "^0.10.0 || ^0.11.0 || ^1.0.0"
```
Comment (ascorbic) [inline packages/remix-netlify/package.json:21]:
```suggestion
    "@netlify/functions": "^1.0.0"
```

The devDependency should be the latest version

**Comment by MichaelDeBoey** [comment]
@donavon Could you please rebase your branch onto latest `dev`?

**Review by MichaelDeBoey — APPROVED** [review]

**Comment by segheysens** [comment]
@chaance @mcansh I think that not having this change in the Remix peerDependencies for the Netlify deployment target breaks [this project](https://github.com/netlify/remix-edge-template/issues/35). Is there anything else you need to approve this PR?

**Review by kentcdodds — APPROVED** [review]
I'm almost always in favor of supporting peer dep version ranges that are as generous as possible.

Thanks!

**Pull request merged**